internal/util/logger/logger.go

```diff
@@ -73,6 +73,18 @@ type LogSink struct {
 	depth int
 }
 
+func init() {
+	// Global-level definitions for zerolog
+	zerolog.MessageFieldName = messageFieldName
+	zerolog.TimestampFieldName = timestampFieldName
+	zerolog.TimeFieldFormat = time.RFC3339Nano
+	zerolog.CallerMarshalFunc = zerologCallerMarshalFunc
+	zerolog.LevelFieldMarshalFunc = zerologLevelFieldMarshalFunc
+
+	// Set zerolog global level to the most verbose level
+	zerolog.SetGlobalLevel(getZerologLevelFromLogrVerbosity(VerboseLevel))
```

Comment:
Confirmed that `getZerologLevelFromLogrVerbosity()` just uses the global `logrVerbosityToZerologLevelMap` which is defined prior to this `Init` function executing according to Golang order of initialization (`var`s defined before `Init` called)